### PR TITLE
enhancements/etcd/cluster-etcd-operator: update render command

### PR DESCRIPTION
**Pull request by hexfusion**
Added more details on the render command for cluster-etcd-operator.

### CODE REVIEW

File: enhancements/etcd/cluster-etcd-operator.md [hunk 1]
```diff
@@ -180,6 +180,20 @@ This controller shapes the static pod manifest itself.  It has some unique featu
     It waits until it is able to know that it is in the member list before trying to start.
     When it does this, it is able to determine the member list to use to launch.
 
+### Render Command
+Populates the static pod manifest used on the bootstrap node and other resource dependencies during the cluster bootstrap.
+
+#### File Reads
+1.) cluster-network-file - to determine the `ClusterCIDR` and `ServiceCIDR`. These values are used to conclude if the cluster is
+    single stack.
```
Comment (deads2k) [inline enhancements/etcd/cluster-etcd-operator.md]: list choices

1. single-stack ipv4
2. single-stack ipv6
3. dual-stack ipv4
4. dual-stack ipv6

indicate which work today and which don't and how you make the choice

File: enhancements/etcd/cluster-etcd-operator.md [hunk 2]
```diff
@@ -180,6 +180,20 @@ This controller shapes the static pod manifest itself.  It has some unique featu
...
+2.) cluster-config-file - checks the declared `MachineCIDR` of the cluster from the install-config. Render uses the `MachineCIDR`
+    of the cluster to validate which IP address on the bootstrap interfaces is the BootstrapIP by making sure it is on the same
+    network as the `MachineCIDR`.
+
+#### File Writes
```
Comment (deads2k) [inline enhancements/etcd/cluster-etcd-operator.md]: add a debt section for pieces we need to move from their existing locations to here.  I think there are some MCO and installer rendered bits

File: enhancements/etcd/cluster-etcd-operator.md (hunk 1 above)
Comment (danwinship) [reply enhancements/etcd/cluster-etcd-operator.md]: "dual-stack" means both ipv4 and ipv6 so there are only 3 choices.

I guess in theory "dual stack with ipv4 cidrs listed first" and "dual stack with ipv6 cidrs listed first" might behave slightly differently at the cluster level but I don't think it matters for etcd.

**Comment by hexfusion** [comment]
@danwinship @deads2k updated per notes

**Comment by russellb** [comment]
@hexfusion is everything in here still current?  It looks like this was just a minor update that fell through the cracks and should just be merged, as long as nothing major has changed invalidating these changes.

**Comment by hexfusion** [comment]
@russellb no this needs updated I will make sure we get that taken care of.

**Comment by hexfusion** [comment]
tracking as part of https://bugzilla.redhat.com/show_bug.cgi?id=1843528

**Pull request closed without merging**